pulsar-testclient/src/main/java/org/apache/pulsar/testclient/DefaultMessageFormatter.java

```diff
@@ -81,21 +81,18 @@ private float _getFloatValue(float size) {
 
     private String getStringValue(float size) {
         int s = (int) size;
-        if (size == 0) {
```

Comment:
It seems that it's a default size for error situation.
Maybe it's better not change it if you are absolute sure?

Reply:
It seems that the `size` is not used later. It looks like the intent of this method is to generate the random string of default size 20 when 0 is passed in. I think the value here should be assigned to the `s` instead of the `size`.

Reply:
That's the test-performance-producer for `%5s` stands for 5 length random string. The command is already widely use in performance tests. For people who want to use length for 20, they can use `%20s`